themes/classic/templates/checkout/order-confirmation.tpl

```diff
@@ -57,7 +57,9 @@
           <div id="order-details" class="col-md-4">
             <h3 class="h3 card-title">{l s='Order details' d='Shop.Theme.Checkout'}:</h3>
             <ul>
-              <li>{l s='Order reference: %reference%' d='Shop.Theme.Checkout' sprintf=['%reference%' => $order.details.reference]}</li>
+              <li>Order reference:
+                <span id="order-reference-value">{l s='%reference%' d='Shop.Theme.Checkout' sprintf=['%reference%' => $order.details.reference]}</span>
+              </li>
```

Comment:
Please don't do that : `Order reference:` is no more translated

Reply:
Ping @Julievrz

Reply:
We need to remove the reference from the wording.

Reply:
You mean creating a string "Order reference:" in Shop.Theme.Checkout and get rid of the "%reference%" in the existing string "Order reference: %reference%"?

Reply:
I have a new solution the .tpl is now as before :+1: